api.js

```diff
@@ -40,3 +40,9 @@ export function deleteComment(id) {
     `https://nc-news-by-amir.onrender.com/api/comments/${id}`
   );
 }
+
+export function getArticlesByTopic(topic) {
+  return axios.get(
+    `https://nc-news-by-amir.onrender.com/api/articles?topic=${topic}`
```

Comment:
Rather than building a new function here, you should be able to reuse the previous getArticles function 👍 Take a look at Axios Params and how this could help.